H/HelFEM/build_tarballs.jl

```diff
@@ -20,20 +22,16 @@ else
     OPENBLAS="${libdir}/libopenblas.${dlext}"
 fi
 
-# On i686, it seems that we need to explicitly tell GCC that we want SSE2. Refs:
-#   https://github.com/JuliaLang/julia/blob/v1.4.1/src/atomics.h#L8-L10
-#   https://stackoverflow.com/questions/16410149/error-sse2-instruction-set-not-enabled-when-including-emmintrin-h
-if [[ "${target}" == i686* ]]; then
-    export CXXFLAGS="-msse -msse2"
-fi
-
 # Compile libhelfem as a static library
 cd ${WORKSPACE}/srcdir/HelFEM/
 cmake \
-    -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DHELFEM_BINARIES=OFF \
+    -DHELFEM_FIND_DEPS=ON \
     -DUSE_OPENMP=OFF \
-    -DHELFEM_BINARIES=OFF -DHELFEM_FIND_DEPS=ON \
     -B build/ -S .
```

Comment:
I took the liberty of sorting these arguments alphabetically and generally aligning them with other JLLs, in the hope that this will help a bit with future maintenance.